guide/source/2.7.4-migration.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Migrating to Meteor 2.7.4
+description: How to migrate your application to Meteor 2.7.4.
+---
+
+Meteor `2.7.4` introduce the new MongoDB Package Async API. For a complete breakdown of the changes, please refer to the [changelog](http://docs.meteor.com/changelog.html).
+
+For this new async API, we have new methods like `findOneAsync`, which behaves exactly like the `findOne` method, but it now returns a promise that needs to be solved to get the data.
```

Comment:
> [...] returns a promise that needs to be **solved** to get the data.

I guess it was supposed to say "resolved"? If not, it sounds a little too adventurous for a change log :stuck_out_tongue:

Reply:
I agree